core-ui/src/shared/components/MonacoEditorESM/Editor.scss

```diff
@@ -1,9 +1,8 @@
 .resource-form__wrapper {
```

Comment:
whoever wrote this file didn't scope these rules for just the editor, so we're changing all wrappers in all the forms. Perhaps we could add scoping while we're here? and make sure we don't break anything else along the way?
or if not, create a separate task for this - we're having too many cases of similar situations.